### PR TITLE
Feature/groups

### PR DESCRIPTION
**Pull request by JekxDevil**
group feature added.
group list display to choose members need to be refactored in vuetify.
backend may need changes, see `TODO` notes related.

### CODE REVIEW

**Comment by micheledallerive** [comment]
The chats are not sorted properly: if I have normal chats and group chats, the group chats are always at the bottom of the list.

**Comment by micheledallerive** [comment]
The messages are not received right after the group chat is created. You need to refresh the page in order to make the messages work.

**Comment by micheledallerive** [comment]
It is hard to understand what's the message and what is the sender (maybe adding the class `font-weight-bold` to the sender element class could help)

**Comment by micheledallerive** [comment]
Tomorrow I am gonna refactor the group creation frontend page.

**Comment by JekxDevil** [comment]
> It is hard to understand what's the message and what is the sender (maybe adding the class `font-weight-bold` to the sender element class could help)

fixed by @micheledallerive [7882fda](https://github.com/ogs-at-usi/handshake/pull/82/commits/7882fda4884129381e181f1868f27d8bea2f520c).

**Comment by JekxDevil** [comment]
> The messages are not received right after the group chat is created. You need to refresh the page in order to make the messages work.

fixed by @JekxDevil [bfc1878](https://github.com/ogs-at-usi/handshake/pull/82/commits/bfc1878e6e441e0a6d79a549ccb4319514f15130).

File: client/src/plugins/api-client.js [hunk 1]
```diff
@@ -56,6 +56,24 @@ class ApiClient {
     return this.axiosInstance.post('/api/chats', { otherId });
   }
 
+  /**
+   * create a group with the logged user and the select user ids list given a title.
+   * @param group {Group} The group to create
+   */
+  createGroup(group) {
+    console.log('api create group:', group);
```
Comment (micheledallerive) [inline client/src/plugins/api-client.js:64]: remove

**Review by micheledallerive — CHANGES_REQUESTED** [review]

File: client/src/plugins/api-client.js (hunk 1 above)
Comment (JekxDevil) [reply client/src/plugins/api-client.js]: fixed [eded274](https://github.com/ogs-at-usi/handshake/pull/82/commits/eded274295911c696714c3f8cf0046e8a283fe0a).

**Review by micheledallerive — APPROVED** [review]

**Review by DonnaDona — APPROVED** [review]
:)

**Pull request merged**